reseq_ros2/launch/reseq_core_launch.py

```diff
@@ -0,0 +1,110 @@
+
+    # check if it is can
+    if config['canbus']['channel'].startswith("can"):
+        core_nodes.append(0)
+
+    launch_config.append(Node(
+                package='reseq_ros2',
+                executable='communication',
+                name='communication',
+                parameters=[{
+                    'can_channel': config['canbus']['channel'],
+                    'modules': addresses,
+                    'joints': joints,
+                    'end_effector': endEffector
+                }]))
+    launch_config.append(Node(
+            package='reseq_ros2',
+            executable='agevar',
+            name='agevar',
+            parameters=[{
+                'a': config['agevar_consts']['a'],
+                'b': config['agevar_consts']['b'],
+                'd': config['agevar_consts']['d'],
+                'r_eq': config['agevar_consts']['r_eq'],
+                'modules': addresses,
+                'joints': joints,
+                'end_effector': endEffector
+            }]))
+    launch_config.append(Node(
+            package='reseq_ros2',
+            executable='scaler',
+            name='scaler',
+            parameters=[{
+                'r_linear_vel': config['scaler_consts']['r_linear_vel'],
+                'r_inverse_radius': config['scaler_consts']['r_inverse_radius'],
+                'r_pitch_vel': config['scaler_consts']['r_pitch_vel'],
+                'r_head_pitch_vel': config['scaler_consts']['r_head_pitch_vel'],
+                'r_head_roll_vel': config['scaler_consts']['r_head_roll_vel'],
+            }]))
+
+    if config['version'] == 'mk1':
+        launch_config.append(Node(
+                package='reseq_ros2',
+                executable='enea',
+                name='enea',
+                parameters=[{
+                    'pitch': config['enea_consts']['i_pitch'],
+                    'head_pitch': config['enea_consts']['i_head_pitch'],
+                    'head_roll': config['enea_consts']['i_head_roll'],
+                    'servo_speed': config['enea_consts']['servo_speed'],
+                    'r_pitch': config['enea_consts']['r_pitch'],
+                    'r_head_pitch': config['enea_consts']['r_head_pitch'],
+                    'r_head_roll': config['enea_consts']['r_head_roll'],
+                    'pitch_conv': config['enea_consts']['pitch_conv'],
+                    'end_effector': endEffector
+                }]))
+
+    # all other reseq_core nodes are core nodes
+    core_nodes += list(range(1, len(launch_config)))
+
+    event_handlers = []
+    # create list of event handlers for each core node
+    for el in core_nodes:
+        event_handlers.append(
+            RegisterEventHandler(
+                event_handler=OnProcessExit(
+                    target_action = launch_config[el],
+                    on_exit=[
+                        LogInfo(msg=f'---------------------{launch_config[el].name} exited, system ends'),
```

Comment:
Two things:
- I do not think that using LogInfo is correct, since it is a FATAL error and not an info
- In the message I read the following message
```
[INFO] [launch.user]: ---------------------None exited, system ends
```
(in my opinion the message can also be `Core node exited` if it is easier)